app/view/tree/LayerTreeController.js

```diff
@@ -264,9 +266,16 @@ Ext.define('MoMo.admin.view.grid.LayerTreeController', {
                             record.get('text')),
             function (btn) {
                 if (btn === 'yes') {
-                    record.erase();
+                    record.erase({
+                        failure: function() {
+                            Ext.toast(
+                                viewModel.get('deleteTreeItemFailed'),
+                                null, 'b');
+                            // TODO: howto undo the store removal of the record?
```

Comment:
Reload and start from scratch? Just a wild guess, of course.

Reply:
tried that, failed hard...